flink-python/pyflink/datastream/data_stream.py

```diff
@@ -1403,6 +1403,50 @@ def reduce(self,
                                             reducing_state_descriptor,
                                             output_type)
 
+    def aggregate(self,
+                  aggregate_function: AggregateFunction,
+                  window_function: Union[WindowFunction, ProcessWindowFunction] = None,
+                  accumulator_type: TypeInformation = None,
+                  result_type: TypeInformation = None) -> DataStream:
```

Comment:
```suggestion
                  output_type: TypeInformation = None) -> DataStream:
```

Keep it consistent with the other methods.